go.mod

```diff
@@ -1,3 +1,7 @@
 module github.com/nukosuke/go-zendesk/v0.3
 
-require github.com/golang/mock v1.3.1
+require (
+	github.com/golang/mock v1.3.1
+	github.com/google/go-querystring v1.0.0
+	github.com/nukosuke/go-zendesk v0.2.0
```

Comment:
where is this getting pulled in? I don't feel like we actually depend on the 0.2.0 client at this point.

Reply:
Sorry,
fixed at 730fcaf and `go mod tidy` to remove unused dependencies from go.sum